src/index.html

```diff
@@ -12,7 +12,42 @@
       href="styles/index.scss"
     />
```

Comment:
You should link to the compiled CSS file, not the SCSS file. Please update the href attribute to 'styles/index.css'.